app/helpers/application_helper.rb

```diff
@@ -40,15 +40,20 @@ def policies_areas_data_container
     Policy.areas.map { |policy| [policy.name, policy.id] }
   end
 
-  # Re-orders the data container such that +selected+ ones appear first.
+  # Re-orders the data container such that +selected+ ones appear first.
+  # and both in alphabetical order
   def prioritise_data_container(unprioritised_container, selected)
```

Comment:
`DataForSelectHelper`, with all of these data_container methods chucked in there for now?